core/src/main/java/io/github/kaktushose/jdac/dispatching/events/Event.java

```diff
@@ -112,7 +112,7 @@ public MessageResolver messageResolver() {
     ///
     /// @return the localized message or the key if not found
     public String localize(String key, Entry... placeholders) {
```

Comment:
should we also replace this in favor of a resolve method that uses MessageResolver?